en-US.json

```diff
@@ -1032,7 +1032,7 @@
 
 	"FAQ_19_H1": "Why is Stremio taking disk space?",
 	"FAQ_19_P1": "Stremio has a <b>local cache</b> to temporarily keep the video files you stream.",
-	"FAQ_19_P2": "This has two benefits: first, it allows Stremio to pre-buffer the video ahead so as to ensure smoother streaming. Second, it keeps the videos in the local cache so you can continue watching them offline.",
+	"FAQ_19_P2": "This has two benefits: first, it allows Stremio to buffer the video to ensure smoother streaming. Second, it keeps the videos in the local cache so you can continue watching them offline.",
```

Comment:
"pre-buffering" is the initial buffering phase, it is a word and this sentence was already correct.